.github/workflows/blank.yml

```diff
@@ -95,6 +95,12 @@ jobs:
           script: |
             mkdir -p /var/www/csr/stage/
             echo '${{ secrets.DEPLOY_CONFIG }}' > /var/www/csr/stage/config.json
+            sudo rm -rf /etc/systemd/system/stage.csr.service.d
+            sudo EDITOR='tee' systemctl edit stage.csr.service <<< '[Service]
```

Comment:
Is it possible to just overwrite this file like 
`echo 'filecontent' > service_file`?
`EDITOR='tee' systemctl edit` looks complicated.
I know it's mentioned in stackoverflow link I sent earlier, but the main idea from that link was `Environment=`, not the exact process of how the file was edited.

Reply:
In the current case we will create dir /etc/systemd/system/stage.csr.service.d and file owerride.conf, which will contains our env vars. By the way, I tested different approaches - this one works fine

Reply:
You are right @antelman107! I will try to make the flow more simple